ludwig/train.py

```diff
@@ -153,6 +154,10 @@ def train_cli(
 
     :return: (`None`)
     """
+    if HYPEROPT in config:
```

Comment:
this runs on the train cli. It should also be run in the experiment cli.
Also, both train and experiment API should have a simila waring, but without the blocking yes/no question, just a warning.